faq.md

```diff
@@ -113,6 +113,45 @@ testing.
 If you have an entire directory that you want to "exclude" from REUSE compliance
 testing, you can [use a DEP5 file](#bulk-license).
 
+## How do I exclude certain files from REUSE compliance testing? {#exclude-lines}
+
+Some files may contain commands or documentation that confuse the REUSE helper tool:
```

Comment:
Answer the question first, provide context afterwards.

Reply:
Good point. I've reordered the answer a bit.